Commencement.Mvc/Content/mainCal.css

```diff
@@ -1,4 +1,4 @@
-﻿@charset "UTF-8";
```

Comment:
@jSylvestre is this mailCal.css file being used anywhere?  If so, want to rename to something better?